_includes/head.njk

```diff
@@ -9,75 +10,106 @@
+  <meta
+    name="fediverse:creator"
+    content="{{ metadata.socialHandles.get('Mastodon') }}"
+  />
 
   {# Social: Open Graph tags #}
-  <meta property="og:locale" content="en_US">
-  <meta property="og:title" content="{{ headTitle }}">
-  <meta property="og:description" content="{{ headDescription }}">
-  <meta property="og:logo" content="{{ metadata.url }}{{ metadata.logoImage }}">
-  <meta property="og:type" content={{ headOGType }}>
-  <meta property="og:url" content={{ metadata.url }}{{ page.url.slice(1) }}>
-  <meta property="og:site_name" content={{ metadata.title }}>
+  <meta property="og:locale" content="en_US" />
+  <meta property="og:title" content="{{ headTitle }}" />
+  <meta property="og:description" content="{{ headDescription }}" />
+  <meta
+    property="og:logo"
+    content="{{ metadata.url }}{{ metadata.logoImage }}"
+  />
+  <meta property="og:type" content="{{ headOGType }}" />
+  <meta property="og:url" content="{{ metadata.url }}{{ page.url.slice(1) }}" />
+  <meta property="og:site_name" content="{{ metadata.title }}" />
   {%- if largeImage %}
-  <meta property="og:image" content="{{ metadata.url }}{{ largeImage }}">
+    <meta property="og:image" content="{{ metadata.url }}{{ largeImage }}" />
   {%- elif image %}
-  <meta property="og:image" content="{{ metadata.url }}{{ image }}">
+    <meta property="og:image" content="{{ metadata.url }}{{ image }}" />
   {%- else %}
-  <meta property="og:image" content="{{ metadata.url }}{{ metadata.titleImage }}">
+    <meta
+      property="og:image"
+      content="{{ metadata.url }}{{ metadata.titleImage }}"
+    />
   {%- endif %}
-  <meta property="og:image:alt" content="{{ headImageAlt }}">
-  <meta property="og:image:type" content="{{ headImageType }}">
+  <meta property="og:image:alt" content="{{ headImageAlt }}" />
+  <meta property="og:image:type" content="{{ headImageType }}" />
 
   {# NOTE: All common CSS used throughout the site are added here. In dev they are created as links, in prod all CSS is concatenated and inlined for a small performance boost. #}
   {%- set partials %}
-      base/reset,
-      base/base,
-      base/utils,
-      components/button,
-      components/theme-toggle,
-      layouts/header,
-      layouts/footer
-    {% endset %}
+    base/reset, base/base, base/utils, components/button,
+    components/theme-toggle, layouts/header, layouts/footer
+  {% endset %}
   {% include "styles.njk" %}
 
-  <script>
-    {# NOTE: prevents a flash of un-themed content. See theme-toggle.js for related code. #}
-    const theme = localStorage.getItem("theme");
-    if (theme) {
-      document.documentElement.dataset.theme = theme;
-    }
-  </script>
+  {# NOTE: prevents a flash of un-themed content. See theme-toggle.js for related code. #}
+  {# prettier-ignore-start #}
+  <script>const theme=localStorage.getItem("theme");if(theme){document.documentElement.dataset.theme=theme}</script>
+  {# prettier-ignore-end #}
```

Comment:
NOTE: manually minified this code to prevent the line breaks in the minified html files